app/pdfs/certificate_pdf.rb

```diff
@@ -2,47 +2,73 @@
 
 class CertificatePDF < Prawn::Document
   attr_accessor :path
+  attr_reader :certificate
 
   PDF_OPTIONS = {
     page_size:   'A5',
     page_layout: :landscape,
     margin:      [30, 75]
-  }
+  }.freeze
```

Comment:
This is a question: why using `freeze`? I'm not questioning its usage, just curious.